packages/medusa/src/strategies/batch-jobs/product/utils.ts

```diff
@@ -54,12 +56,14 @@ export function transformVariantData(
 
   Object.keys(productData).forEach((k) => {
     const key = k.split("variant.")[1]
+
+    set(ret, key, productData[k])
```

Comment:
question: Line 60 + 61 are duplicates. We should be able to safely delete one of them no?